src/spu/src/services/public/stream_fetch.rs

```diff
@@ -449,10 +449,12 @@ pub mod publishers {
         }
 
         // get next stream id
+        #[instrument(skip(self))]
         fn next_stream_id(&self) -> u32 {
             self.stream_id.fetch_add(1, SeqCst)
         }
 
+        #[instrument(skip(self))]
```

Comment:
remove

Reply:
The original reason I put this in this area because of the async mutexes involved. I was thinking it would be a good way to spot any problems if we get any high lock contention or anything like that.